src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -434,11 +448,15 @@ EOF
              ncks -4 -h -v time_stamp timestamp.nc4 -A ${EXPID}.${ThisCol}.$YYYY$MM$DD.nc4
              /bin/rm timestamp.cdl
              /bin/rm timestamp.nc4
+             # delete sub-daily files
```

Comment:
check two things here:
1) does the desired daily concatenated file exist
2) is it non-zero size
if so, proceed with deletion of sub-daily files, otherwise leave sub-daily files in place and move on